src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -550,7 +557,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             selectedAsset: assetMetadata,
         }, async () => {
             await this.onAssetMetadataChanged(assetMetadata);
-            await this.props.actions.saveProject(this.props.project);
+            await this.props.actions.saveProject(this.props.project, false);
```

Comment:
don't need to save to field.json. only need to update projects lastvisited

Reply:
also gets executed before updateProjectTagsFromFiles in editorpage's componentDidMounted and overwrites field.json with current projects tags